src/server/authserver/authserver.conf.dist

```diff
@@ -157,6 +157,15 @@ SourceDirectory  = ""
 
 MySQLExecutable = ""
 
+#
+#    IPLocationFile
+#        Description: The path to your IP2Location database CSV file.
+#        Example:     "C:/Trinity/IP2Location.csv"
+#                     "/home/trinity/IP2Location.csv"
+#        Default:     "" - (Disabled)
```

Comment:
Won't the default be "current directory" or "current core directory" here?
```
#        Default:     IPLocationFile = "." (current core directory)
```
(this is just my suggestion, the format may be slightly inaccurate)